EIPS/eip-3643.md

```diff
@@ -36,7 +32,7 @@ The functions described by these interfaces vary and allow the appropriate users
 The following requirements have been compiled following discussions with parties across financial institutions that are looking to issue securities on a DLT infrastructure such as ethereum.
 
 - **MUST** be [ERC-20](./eip-20.md) compatible.
-- **MUST** be used in combination with an Identification system onchain ([ONCHAINID](https://github.com/onchain-id/solidity))
+- **MUST** be used in combination with an onchain Identity system
 - **MUST** be able to apply any rule of compliance that is required by the regulator or by the token issuer (about the factors of eligibility of an identity or about the rules of the token itself)
 - **MUST** have a standard interface to pre-check if a transfer is going to pass or fail before sending it to the blockchain
 - **MUST** have a recovery system in case an investor loses access to his private key
```

Comment:
Because this isn't the specification section, you can't add requirements on implementations here. From my quick reading, it seems like these are requirements on the standard itself? To avoid confusion, I'd recommend not using the capitalized `MUST` notation, and maybe rewording this into a paragraph instead of bullet points.